### PR TITLE
Add possibility to set parameters for project creation

### PR DESCRIPTION
**Pull request by Springrbua**
Closes angelozerr/angular2-eclipse#31

It is now possible to set following parameters in the "New Angular2 Project"-Wizard:
- skip-npm
- skip-git
- skip-tests
- skip-commit 
- source-dir
- style
- prefix
- routing
- inline-style
- inline-template

### CODE REVIEW

File: ts.eclipse.ide.angular2.cli/src/ts/eclipse/ide/angular2/internal/cli/wizards/NewAngular2ProjectParamsWizardPage.java [hunk 1]
```diff
@@ -0,0 +1,386 @@
+package ts.eclipse.ide.angular2.internal.cli.wizards;
```
Comment (angelozerr) [inline ts.eclipse.ide.angular2.cli/src/ts/eclipse/ide/angular2/internal/cli/wizards/NewAngular2ProjectParamsWizardPage.java]: Please add license + your name.

File: ts.eclipse.ide.angular2.cli/src/ts/eclipse/ide/angular2/internal/cli/wizards/NewAngular2ProjectParamsWizardPage.java [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Combo;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Event;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Listener;
+import org.eclipse.swt.widgets.Text;
+
+import ts.eclipse.ide.angular2.internal.cli.AngularCLIMessages;
+
```
Comment (angelozerr) [inline ts.eclipse.ide.angular2.cli/src/ts/eclipse/ide/angular2/internal/cli/wizards/NewAngular2ProjectParamsWizardPage.java]: Please add comments which explains class.

File: ts.eclipse.ide.angular2.cli/src/ts/eclipse/ide/angular2/internal/cli/wizards/NewAngular2ProjectParamsWizardPage.java (hunk 1 above)
Comment (Springrbua) [inline ts.eclipse.ide.angular2.cli/src/ts/eclipse/ide/angular2/internal/cli/wizards/NewAngular2ProjectParamsWizardPage.java]: Thanks for reviewing, I'll add that as soon as possible.

Comment (Springrbua) [inline ts.eclipse.ide.angular2.cli/src/ts/eclipse/ide/angular2/internal/cli/wizards/NewAngular2ProjectParamsWizardPage.java]: @angelozerr Can I add Github nickname? Is the E-Mail required to?

Comment (angelozerr) [inline ts.eclipse.ide.angular2.cli/src/ts/eclipse/ide/angular2/internal/cli/wizards/NewAngular2ProjectParamsWizardPage.java]: Yes sure, email  is not requied.

**Comment by angelozerr** [comment]
Thanks @Springrbua !

**Pull request merged**